core-build/gulp-core-build/src/index.ts

```diff
@@ -120,6 +120,16 @@ export function getConfig(): IBuildConfig {
   return _buildConfig;
 }
 
+const cleanFlagTask: CleanTask = new CleanTask();
+let hasCleanFlagTaskRun: boolean = false;
+cleanFlagTask.isEnabled = (buildConfig: IBuildConfig) => {
+  // tslint:disable-next-line:no-string-literal
+  const shouldRun: boolean = (!!buildConfig.args['clean'] || !!buildConfig.args['c'])
+    && hasCleanFlagTaskRun === false;
+  hasCleanFlagTaskRun = true;
```

Comment:
>hasCleanFlagTaskRun [](start = 2, length = 19)

What does this do? Seems like the clean task's `isEnabled` should always return true if the `--clean` flag was specified.